sdk/textanalytics/ai-text-analytics/samples-dev/customText.ts

```diff
@@ -0,0 +1,135 @@
+  "I need to take my cat to the veterinarian.",
+  "The employee's SSN is 555-55-5555.",
+  "We went to Contoso Steakhouse located at midtown NYC last week for a dinner party, and we adore the spot! They provide marvelous food and they have a great menu. The chief cook happens to be the owner (I think his name is John Doe) and he is super nice, coming out of the kitchen and greeted us all. We enjoyed very much dining in the place! The Sirloin steak I ordered was tender and juicy, and the place was impeccably clean. You can even pre-order from their online menu at www.contososteakhouse.com, call 312-555-0176 or send email to order@contososteakhouse.com! The only complaint I have is the food didn't come fast enough. Overall I highly recommend it!"
+];
+
+export async function main() {
+  console.log("== Analyze Sample ==");
+
+  const client = new TextAnalyticsClient(endpoint, new AzureKeyCredential(apiKey));
+
+  const actions: TextAnalyticsActions = {
+    recognizeCustomEntitiesActions: [
+      {
+        projectName: process.env["RECOGNIZE_CUSTOM_ENTITIES_PROJECT_NAME"]!,
+        deploymentName: process.env["RECOGNIZE_CUSTOM_ENTITIES_DEPLOYMENT_NAME"]!
+      }
+    ],
+    classifyDocumentSingleCategoryActions: [
+      {
+        projectName: process.env["CLASSIFY_DOCUMENT_SINGLE_CATEGORY_PROJECT_NAME"]!,
+        deploymentName: process.env["CLASSIFY_DOCUMENT_SINGLE_CATEGORY_DEPLOYMENT_NAME"]!
+      }
+    ],
+    classifyDocumentMultiCategoriesActions: [
+      {
+        projectName: process.env["CLASSIFY_DOCUMENT_MULTIPLE_CATEGORY_PROJECT_NAME"]!,
+        deploymentName: process.env["CLASSIFY_DOCUMENT_MULTIPLE_CATEGORY_DEPLOYMENT_NAME"]!
+      }
+    ]
+  };
+  const poller = await client.beginAnalyzeActions(documents, actions, "en", {
+    includeStatistics: true
+  });
+
+  poller.onProgress(() => {
+    console.log(
+      `Number of actions still in progress: ${poller.getOperationState().actionsInProgressCount}`
+    );
+  });
+
+  console.log(`The analyze actions operation created on ${poller.getOperationState().createdOn}`);
+
+  console.log(
+    `The analyze actions operation results will expire on ${poller.getOperationState().expiresOn}`
+  );
+
+  const resultPages = await poller.pollUntilDone();
+
+  for await (const page of resultPages) {
+    const customEntitiesAction = page.recognizeCustomEntitiesResults[0];
+    if (!customEntitiesAction.error) {
+      for (const doc of customEntitiesAction.results) {
+        console.log(`- Document ${doc.id}`);
+        if (!doc.error) {
+          console.log("\tEntities:");
+          for (const entity of doc.entities) {
+            console.log(`\t- Entity ${entity.text} of type ${entity.category}`);
+          }
+        } else {
+          console.error("\tError:", doc.error);
+        }
+      }
+      console.log("Action statistics: ");
+      console.log(JSON.stringify(customEntitiesAction.results.statistics));
```

Comment:
If you write JSON to console, I would pretty print it.

```suggestion
      console.log(JSON.stringify(customEntitiesAction.results.statistics, null, 2));
```